assets/js/base/components/reviews/review-list/index.tsx

```diff
@@ -2,26 +2,28 @@
  * External dependencies
  */
 import { getSetting } from '@woocommerce/settings';
-import type { BlockAttributes } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import ReviewListItem from '../review-list-item';
 import type { Review } from '../types';
 import './style.scss';
-
+import { ReviewBlockAttributes } from '../../../../blocks/reviews/attributes';
```

Comment:
Long relative import here.

Reply:
Resolved in https://github.com/woocommerce/woocommerce-blocks/pull/10677/commits/f20885e4c2180c190b960ad04a02e943fbccd112